Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -4274,25 +4272,19 @@
     "Context": {
       "ReferenceDateTime": "2018-04-07T00:00:00"
     },
-    "Comment": "FORMAT not properly handled.",	
-    "NotSupported": "dotnet, javascript, python, java",
+    "NotSupported": "javascript, python, java",
     "Results": [
       {
-        "Text": "1.º de julio",
+        "Text": "1.º de julio, 17",
```

Comment:
This is not correct. In the input sentence, 17 should not match.

Reply:
yes, but since in English "17" is also extracted in the corresponding case (July 1st, 17), I have not modified this behaviour. Only, when "17" is an ordinal it is not considered part of the extraction in English (and now in Spanish).
I will modify both languages behaviour to exclude "17" also in this case.

Reply:
Thanks for bringing attention to this!